prime-router/src/main/resources/db/migration/V71__extend_character_limits.sql

```diff
@@ -0,0 +1,178 @@
+/*
+ * The Flyway tool applies this migration to create the database.
+ *
+ * Follow this style guide https://about.gitlab.com/handbook/business-ops/data-team/platform/sql-style-guide/
+ *
+ * Copy a version of this comment into the next migration
+ *
+ */
+
+/*
+ * Extend length of character limited fields that are in danger of being exceeded in production.
+ */
+
+/* Columns used by views cannot be altered, so we have to drop the view first. */
+
+/* Clean up any existing structures for the view that exist now */
+DROP INDEX IF EXISTS idx_vw_livid_manufacturer;
+DROP INDEX IF EXISTS idx_vw_livid_model;
+DROP INDEX IF EXISTS idx_vw_livid_test_performed_loinc_code;
+DROP INDEX IF EXISTS idx_vw_livid_test_orderd_loinc_code;
+DROP INDEX IF EXISTS idx_vw_livid_otc;
+
+/* Remove the view */
+DROP MATERIALIZED VIEW IF EXISTS vw_livd_table;
+
+
+ALTER TABLE action
+ALTER COLUMN action_params TYPE VARCHAR,
```

Comment:
Please provide the original COLUMN types for each of these with reasoning for using an effectively unlimited field.

Reply:
I think adding original column types is a good idea for documentation purposes. @jack-h-wang the reasoning that applies to most of these is "Previous limit was arbitrary and there is not a solid application requirement for limiting this field to a certain length". 

The only one I think that does not have non-arbitrary limits they cannot exceed lest they be invalid and could have a length constraint is `sender_ip` (39 characters to support IPv6), but I think we can just leave is without the limit like you have here.

Reply:
The migration now includes the original types for all fields that are changed. Justification has been added to the beginning comments of the migration.